feat(react-native): live stream components for host and viewer (#1135)
GetStream/stream-video-js: packages/react-native-sdk/docusaurus/docs/reactnative/common-content/ui-components/livestream/viewer-livestream/viewer-leave-stream-button.mdx

> Component to customize the leave stream button on the viewer's end live stream.

Reviewer: ```suggestion
Component to customize the leave stream button for the viewer.
```

Just simplifying the language
